packages/store/src/storage.ts

```diff
@@ -983,7 +983,7 @@ class Storage {
     const localPackage = await this.getPackageManifest({
       name,
       requestOptions,
-      uplinksLook: false,
```

Comment:
here `false` is intended, check the variable name, `localPackage`

Reply:
I'll revert it.